_posts/2022-11-10-international-show-n-tell-2022.md

```diff
@@ -0,0 +1,58 @@
+  - showntell
+  - javascript
+  - nodejs
+  - html
+layout: event.html
+collection: events
+online: true
+venue:
+  name: Spaces
+  link: https://www.spacesworks.com/brighton/trafalgar-place/
+  location: https://g.page/Mocatta-House-4437?share
+  address: Mocatta House, Trafalgar Pl, BN1 4DU, Brighton
+  latlong: 50.8294399,-0.1443907
+---
+
+The November migration of the world's top web developers to the sunny city of Brighton & Hove can only mean one thing - [FFConf 2022](https://2022.ffconf.org/) is coming. So, it's time for another International Async Show 'n' tell! This year's installment will be _on Thursday 10th November_. The ever popular International show 'n' tell is the perfect opportunity to give a peek at your passion project, try out a lightning talk, or just share what's on your mind.
+
+If you're interested in giving a talk, we'd love to hear from you. It's the perfect opportunity to give a peek at your passion project, try out a lightning talk, or just share what's on your mind. You’ll have five minutes on the night to tell us about something… a project, a library, a technique, a story, or anything else about JavaScript, related tech (HTML, CSS, SVG, etc), or the wider world of web.
+
+If you would like to grab a speaking slot, please leave a comment on the [meetup event][meetup] including:
+
+- Your name
+- Your main website, if you have one
+- Twitter username, if you have one
+- A sentence or two about your talk
+- Any relevant links
+
+For a spot of inspiration, take a look at our International Show n' Tells in [2021][showntell-2021], [2020][showntell-2020], [2019][showntell-2019], [2018][showntell-2018], [2017][showntell-2017], [2016][showntell-2016], [2015][showntell-2015], [2014][showntell-2014], [2013][showntell-2013], [2012][showntell-2012], [2011][showntell-2011] and [2010][showntell-2010].
+
+You will be able to join us in-person at [Spaces](https://www.spacesworks.com/brighton/trafalgar-place/), or online (via Zoom).
+
+If you're thinking of joining us in-person, please sign up to the event on [meetup.com](https://www.meetup.com/async-web-tech-meetup/events/289172125/). We also highly recommend you sign up to the [Async Slack](https://join.slack.com/t/asyncjs/shared_invite/zt-1aguxx86q-XjF_yWcFoJ8fyYYzoqgDaQ) for more info, and updates.
+
+[showntell-2010]: http://asyncjs.com/showntell3/
+[showntell-2011]: http://asyncjs.com/international2011/
+[showntell-2012]: http://asyncjs.com/showntell-2012/
+[showntell-2013]: http://asyncjs.com/showntell-2013/
+[showntell-2014]: http://asyncjs.com/showntell-2014/
+[showntell-2015]: http://asyncjs.com/showntell-2015/
+[showntell-2016]: https://asyncjs.com/international-show-n-tell-2016/
+[showntell-2017]: https://asyncjs.com/international-show-n-tell-2017/
+[showntell-2018]: https://asyncjs.com/international-show-n-tell-2018/
+[showntell-2019]: https://asyncjs.com/international-show-n-tell-2019/
+[showntell-2020]: https://www.meetup.com/async-web-tech-meetup/events/271442327/
```

Comment:
should 2020 and 2021 also link to the Async pages?

2020: https://asyncjs.com/two-dimensional-show-n-tell/
2021: https://asyncjs.com/two-dimensional-show-n-tell-2021/

Reply:
The pages used to contain the comments from the speaker gist, but the last 2 years they've been collected on the Meetup comments.

Reply:
May be worth us looking into the Meetup API to pull comments from there to the async site (in the future).

Reply:
ah good call, I see what you've done there